### DISCUSSION TITLE
Let's mention all features and contributors in our ChangeLog.md

### DISCUSSION THREAD

**Discussion by michaliskambi** [pasdoc/pasdoc#165] [Announcements]
Each time I merge some important feature/bugfix, I add it to our `ChangeLog.md` file in https://github.com/pasdoc/pasdoc/blob/master/ChangeLog.md , at the end of the section _"Upcoming Next Release"_ there. It's an important text IMHO, as it is the primary way how we inform users about the new features, where we send users to more information (like docs for new features).

In keeping it "up to date" as we go, this means we always have a nice changelog, and we're always ready to make a release -- at which point I will use _"Upcoming Next Release"_ for the release notes, turn it into a section _"Version X.Y.Z ..."_, and add new empty _"Upcoming Next Release"_ section. The release notes are used for GitHub release, and at many other places where we post about new PasDoc release.

Everyone: you are most welcome to watch this file (did I remember to mention everything, with proper credits) and help me maintain it (you're welcome to modify `ChangeLog.md` as part of the PR that introduces relevant feature). Add there stuff, add there links, make it better etc :)

**Comment by Fr0sT-Brutal** [comment]
How about a new release?

**Reply by michaliskambi** [reply]
I think yes, I'll make new topic here to be prominent :)
